cmd/jaeger-stack/docker-compose.yml

```diff
@@ -0,0 +1,73 @@
+version: '2'
+services:
+  cassandra:
+    image: cassandra:3.11
+    networks:
+      - jaeger
+    ports:
+      - "7000:7000"
+      - "7001:7001"
+      - "7199:7199"
+      - "9042:9042"
+      - "9160:9160"
+    restart: unless-stopped
+    volumes:
+      - cassandra-data:/var/lib/cassandra
+      - cassandra-logs:/var/log/cassandra
+    logging:
+      options:
+        max-size: 5m
+    environment:
+      MAX_HEAP_SIZE: 512M
+      HEAP_NEWSIZE: 100M
+      CASSANDRA_CLUSTER_NAME: "jaeger"
+      CASSANDRA_DC: "dc1"
+      CASSANDRA_RACK: "rack1"
+      CASSANDRA_ENDPOINT_SNITCH: "GossipingPropertyFileSnitch"
+      CASSANDRA_START_RPC: "true"
+  cassandra-scheme:
+    image: jaegertracing/jaeger-cassandra-schema:0.5
+    networks:
+      - jaeger
+    depends_on:
+      - cassandra
+  jaeger-collector:
+    image: jaegertracing/jaeger-collector:0.5
+    networks:
+      - jaeger
+    ports:
+      - "14267:14267"
+      - "14268:14268"
+    depends_on:
+      - cassandra-scheme
+    restart: unless-stopped
+    command: /go/bin/collector-linux -cassandra.servers=cassandra -cassandra.keyspace=jaeger_v1_dc1 -cassandra.connections-per-host 2
+  jaeger-agent:
+    image: jaegertracing/jaeger-agent:0.5
```

Comment:
think it could be altered to do a live build, https://docs.docker.com/compose/compose-file/compose-file-v2/#build for the image on the test infrastructure, but i'm not 100% aware of all build prerequisites for the jaeger components to build. For the purpose of handing it over to people who are interested in trying out the setup with all components (not the standalone way), i'd rather vote for fixed versions, but that would indeed be followed by regular updates. Given the test run properly, than "latest" wouldn't be an issue. But that's a bit out of scope for me. So probably two docker files? Or even more? There is a stack up and coming with ES, and most likely you won't have the agent on the collectors Infrastructure neither. So probably the jagertracing org would indeed be a better fit.

For production environments i'll soon setup a rancher catalog and add it to the rancher repo anyways.